src/packages/AgentLiveClient.ts

```diff
@@ -0,0 +1,165 @@
+import { AgentEvents } from "../lib/enums/AgentEvents";
+import type { AgentLiveSchema, SpeakModel } from "../lib/types";
+import type { DeepgramClientOptions } from "../lib/types";
+import { AbstractLiveClient } from "./AbstractLiveClient";
+
+export class AgentLiveClient extends AbstractLiveClient {
+  public namespace: string = "agent";
+
+  constructor(options: DeepgramClientOptions, endpoint: string = "/agent") {
+    super(options);
+    /**
+     * According to the docs, this is the correct base URL for the Agent API.
+     * TODO: Make configurable for self-hosted customers.
+     */
+    this.baseUrl = "wss://agent.deepgram.com";
```

Comment:
We should set this as a default in the namespace and let the existing config handler allow overriding it

https://github.com/deepgram/deepgram-js-sdk/blob/7e618010c33334c7603654d0f7f42368e57a6587/src/lib/constants.ts#L47-L49

We'll need another line here like `agent: DEFAULT_GLOBAL_OPTIONS` and define websocket.options.url inside that constant